finans/rapport_includes/moms_rubrik.php

```
@@ -0,0 +1,329 @@
+    if ($dato_til)   $slutdato    = $dato_til;
+
+    $startdato = (int)$startdato;
+    if ($startdato < 10) $startdato = '0'.$startdato;
+    while (!checkdate($startmaaned, $startdato, $startaar)) { $startdato--; if ($startdato < 28) break; }
+    while (!checkdate($slutmaaned,  $slutdato,  $slutaar))  { $slutdato--;  if ($slutdato  < 28) break; }
+    if (strlen((string)$startdato) < 2) $startdato = '0'.$startdato;
+
+    $regnstart = "$startaar-$startmaaned-$startdato";
+    $regnslut  = "$slutaar-$slutmaaned-$slutdato";
+
+    $dim = '';
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$dim'. (undefined)

(UnusedLocalVariable)